src/CanopyContext.js

```diff
@@ -108,11 +110,17 @@ export function CanopyProvider({
   }, []);
 
   useEffect(() => {
-    fetchUserSaplings(saplingURL).then(saplings => {
-      mountSaplingStyles(saplings);
-      mountCurrentSapling(saplings);
-      setUserSaplings(saplings);
-    });
+    if (sessionUser) {
```

Comment:
Not blocking because this might be an obscure feature - but at some point we might want this login functionality to be an option that we can turn on or off.